### DISCUSSION TITLE
Need help about teledrive

### DISCUSSION THREAD

**Discussion by demonfatal** [mgilangjanuar/teledrive#95] [Q&A] [answered]
First thanks for this marvelous app!

I have some questions :
1. Can you tell me if they are a way to mount my teledrive personal folder on my raspberry?
2. If I send file using telegram-upload, how to send them directly in my teledrive folder ? 
3. It is possible to sync a folder like photos and upload new files directly to teledrive?

Thanks for your help!

**Comment by mgilangjanuar** [comment, marked as answer]
For now, it's all not possible and I can't promise anything to you. But, we'll do the best 😄

**Comment by demonfatal** [comment]
Well thank you for your reply.

If I install your teledrive app (with yarn) on my server, what can I do more than using your web version?

**Reply by mgilangjanuar** [reply]
Maybe you can add your own features, a more reliable and private server, etc you like.
